tools/efs-setup-crds

```diff
@@ -1,16 +0,0 @@
-#! /bin/bash
```

Comment:
Has this been made obsolete because we've added this to the helm configs?

Reply:
Jake has other methods.   I set this up in about 20 minutes for some webbinar at some point but (a) it's never been used for real and (b) I think it currently leaves files behind owned as root.   If we ever need it we can just back up in git.